docs/doc_sources/Usage.rst

```diff
@@ -24,3 +24,4 @@ The example notebook linked below can be found within the `docs folder <https://
 .. toctree::
 
   getting_started
+  separatrix_operational_space
```

Comment:
https://github.com/cfs-energy/cfspopcon/pull/12#discussion_r1465270822
Add to a separate section, instead of the example notebook section